cli/src/main/scala/com/codacy/analysis/cli/Main.scala

```diff
@@ -233,8 +234,10 @@ class MainImpl extends CLIApp {
     }
   }
 
-  private def cleanup(directoryOpt: Option[File]): Unit = {
-    val directory = directoryOpt.getOrElse(File.currentWorkingDirectory) / ".codacy.json"
-    directory.delete(swallowIOExceptions = true)
+  private def cleanup(directory: File): Unit = {
+    val codacyJson = directory / ".codacy.json"
```

Comment:
I think `directory / ".codacy.json" delete(swallowIOExceptions = true)` would be cleaner.

Also a comment near the cleanup explaining it is temporary while the tools are migrated to the new version of the seed.